src/Grpc.AspNetCore.Server/Internal/HttpContextSerializationContext.cs

```diff
@@ -39,6 +40,7 @@ internal sealed class HttpContextSerializationContext : SerializationContext
 
         public PipeWriter ResponseBufferWriter { get; set; } = default!;
 
+        [MemberNotNullWhen(true, nameof(_payloadLength))]
```

Comment:
```suggestion
```
It's for nullable _reference_ types, so for `_payloadLength` (as valuetype) it has no effect.

Reply:
It stops the nullable analyzer from complaining about `Nullable<T>.Value`.

Reply:
Ah, correct.

I tried it with VS 17.3.1 and didn't see anything. Now tried it again, and see what you say.
Cool, so please leave it as is.